src/util/expr.h

```diff
@@ -52,14 +61,14 @@ class exprt:public irept
     add(ID_type, _type);
   }
 
-  // returns the type of the expression
+  /// Return the type of the expression
```

Comment:
nitpick: full stop at the end

Reply:
I don't think we usually put a full stop at the end of the return statements.

Reply:
fine with me, but you did that below